.chloggen/2449-fix-ingress.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
This is a correct behaviour of the operator. 

Automatically putting ingress specific annotation should be a new feature

Reply:
I don't think so. Because the Ingress doesn't work otherwise.

Reply:
Take a look to #2449

Reply:
The assumption was that users setup URL rewrite annotations

Reply:
Where is that documented? Because I was not able to find anything related to that.

Reply:
OK. I can revert the annotation, but we still need the other change.

Reply:
The current implementation is broken as enabling ingress results in 404 for all paths.